tests/integrations/jupyter/test_magics.py

```diff
@@ -26,15 +25,6 @@
 RICH_PRE_STYLE = "white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace"
 FREEZE_TIME = "2023-01-01 00:00:00"
 
-# Remove once 3.7 support is dropped
-pytestmark = [
-    pytest.mark.jupyter,
```

Comment:
The comment should have been clearer. We want to keep this mark but we can remove the `skipIf` below.